packages/orbit-components/package.json

```diff
@@ -90,16 +90,16 @@
     "@nanostores/router": "^0.3.1",
     "@playwright/experimental-ct-react": "^1.44.1",
     "@size-limit/webpack-why": "^8.0.0",
-    "@storybook/addon-a11y": "^7.4.6",
-    "@storybook/addon-essentials": "^7.0.23",
-    "@storybook/addon-knobs": "^7.0.2",
-    "@storybook/addon-styling": "^1.3.1",
-    "@storybook/addons": "^7.0.23",
-    "@storybook/blocks": "^7.0.23",
-    "@storybook/react": "^7.0.23",
-    "@storybook/react-webpack5": "^7.0.23",
+    "@storybook/addon-a11y": "^8.1.11",
+    "@storybook/addon-essentials": "^8.1.11",
+    "@storybook/addon-knobs": "8.0.1",
+    "@storybook/addon-styling": "^1.3.7",
```

Comment:
These two addons are deprecated, should we replaced them with their replacements?